tools/jlm-hls/Makefile.sub

```diff
@@ -15,18 +15,19 @@ jlm-hls_LIBS = \
 jlm-hls_EXTRA_LDFLAGS = $(shell $(LLVMCONFIG) --libs core irReader) $(shell $(LLVMCONFIG) --ldflags) $(shell $(LLVMCONFIG) --system-libs)
 
 jlm-hls_EXTRA_LDFLAGS += \
-        -L$(CIRCT_PATH)/lib \
-        -lMLIR \
-        -lCIRCTAnalysisTestPasses \
-        -lCIRCTDependenceAnalysis \
-        -lCIRCTExportFIRRTL \
-        -lCIRCTFIRRTL \
-        -lCIRCTFIRRTLTransforms \
-        -lCIRCTScheduling \
-        -lCIRCTSchedulingAnalysis \
-        -lCIRCTSeq \
-        -lCIRCTSupport \
-        -lCIRCTTransforms \
-        -lCIRCTHW
+	-L$(CIRCT_PATH)/lib \
+	-lMLIR \
+	-lCIRCTAnalysisTestPasses \
+	-lCIRCTDependenceAnalysis \
+	-lCIRCTExportFIRRTL \
+	-lCIRCTFIRRTL \
+	-lCIRCTFIRRTLTransforms \
+	-lCIRCTScheduling \
+	-lCIRCTSchedulingAnalysis \
+	-lCIRCTSeq \
+	-lCIRCTSupport \
+	-lCIRCTTransforms \
+	-lCIRCTHW \
+	${MLIR_LDFLAGS} \
```

Comment:
The same here.

Reply:
This has been discussed earlier and is item 3 in issue https://github.com/phate/jlm/issues/379

Reply:
MLIR_LDFLAGS have been removed.